README.md

```diff
@@ -87,9 +87,20 @@ AutoFixture is available via NuGet:
 * [AutoFixture.Idioms](http://nuget.org/packages/AutoFixture.Idioms)
 * [SemanticComparison](http://nuget.org/packages/SemanticComparison)
 
+### vNext feed
+The artifacts of the next major version are published to the MyGet feed:
+- `https://www.myget.org/F/autofixture/api/v3/index.json` (Visual Studio 2015+)
+- `https://www.myget.org/F/autofixture/api/v2` (Visual Studio 2012+)
+
+You can use this feed to early access and test the next major version of the product.
```

Comment:
Thanks! I rephrased that a few times and didn't know how it's better to say. Indeed, "AutoFixture" looks great 👍